pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -392,7 +392,7 @@ const MemoryRow = ({ memorySize, memorySizeUnit, nodeMaxMemory, recommendedMemor
     if (recommendedMemory && recommendedMemory > memorySize) {
         recommendedMemoryHelpBlock = <p>{cockpit.format(
             _("The selected Operating System has recommended memory $0 $1"),
-            recommendedMemory, memorySizeUnit)}</p>;
+            Math.floor(convertToUnit(recommendedMemory, units.B, memorySizeUnit)), memorySizeUnit)}</p>;
```

Comment:
commit message: "store the recommended resource values in B" .. this is "bytes", right? Also, "when in needed" → "when needed"

Reply:
Yes - and fixed.